src/v2/view-builder/views/email/RequiredFactorEmailView.js

```diff
@@ -39,16 +47,13 @@ const ResendView = View.extend(
 );
 
 const Body = BaseForm.extend(Object.assign(
-
   {
     save: loc('mfa.challenge.verify', 'login'),
+    subtitle:'subtitle',
```

Comment:
Do we need to add a subtitle ?

Reply:
yeah, if leaves empty it don't add the o-form-subtitle class.

Reply:
We can override this behavior in our version of BaseForm

Reply:
but would this add a subtitle to the email verification screen ?

Reply:
It adds, but then we empty the dom and replace with correct title.